packages/ffe-tabs/less/tab.less

```diff
@@ -9,63 +9,77 @@
 // Styleguide ffe-tabs.tab
 
 .ffe-tab {
-    background-color: @ffe-grey-cloud;
-    border: solid 2px @ffe-grey-silver;
-    border-bottom: none;
-    color: @ffe-grey-charcoal;
-    cursor: pointer;
-    display: block;
+    display: flex;
+    background-color: @ffe-farge-hvit;
+    color: @ffe-farge-fjell;
+    padding: @ffe-spacing-xs @ffe-spacing-md;
+    border-radius: 40px;
+    white-space: nowrap;
+    border: 2px solid transparent;
+    margin: @ffe-spacing-2xs;
     font-family: 'SpareBank1-regular', 'MuseoSans-500', arial, sans-serif;
-    font-size: 1rem;
-    outline: none;
-    overflow: hidden;
-    padding: @ffe-spacing-xs @ffe-spacing-lg @ffe-spacing-xs;
-    text-align: center;
-    text-decoration: none;
+    align-self: stretch;
+    justify-content: center;
     transition: all @ffe-transition-duration @ffe-ease;
-    vertical-align: middle;
+    cursor: pointer;
+
     .native & {
         @media (prefers-color-scheme: dark) {
-            background-color: @ffe-grey-charcoal-darkmode;
-            border-color: @ffe-grey-silver-darkmode;
-            color: @ffe-grey-cloud-darkmode;
+            color: @ffe-farge-vann-30;
+            background-color: @ffe-farge-svart;
         }
     }
 
-    &:hover,
-    &:focus,
-    &:active {
-        color: @ffe-blue-royal;
-        background: @ffe-white;
+    &:focus {
```

Comment:
Transition på gif har en litt awkward bevegelse. Det ser ut som om borderen vokser fra knappens størrelse på hvert element som fokuseres,  og resultatet blir en litt hakkete opplevelse:

![Alt Text](https://i.imgur.com/R09vbIM.gif)

En workaround for å få dette til å se litt smoothere ut, som har blitt brukt på knapper tidligere, er å sette box-shadowen på et pseudo-element og transition kun på opacity:

![Alt Text](https://i.imgur.com/sYucsbz.gif)

Kan implementeres med å erstatte linje 33 - 48 med dette:

```
   &::after {
        content: '';
        opacity: 0;
        display: block;
        position: absolute;
        background: transparent;
        border-radius: 1.5rem;
        top: -5px;
        left: -5px;
        right: -5px;
        bottom: -5px;
        box-shadow: 0 0 0 2px @ffe-farge-vann;
        transition: opacity @ffe-transition-duration @ffe-ease;

        .native & {
            box-shadow: 0 0 0 2px @ffe-farge-hvit;
        }
    }

    &:focus::after {
        opacity: 1;
    }

    &:focus:not(:focus-visible)::after {
        opacity: 0;
    }

    &:focus {
        outline: none;
        position: relative;
        overflow: unset;
    }
```

... og legge til `position: relative;` på `.ffe-tab`. :)